common/c/targets/common/integer_math.c

```diff
@@ -0,0 +1,29 @@
+/*
+ *  Copyright 2014-2017 CyberVision, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#include "interger_math.h"
+
```

Comment:
IMHO, it would be better to change this function just to return the sinus of the argument instead of the A*sin(arg).